Kickstarter-iOS/Views/Controllers/SignupViewController.swift

```diff
@@ -95,24 +72,48 @@ internal final class SignupViewController: UIViewController, MFMailComposeViewCo
       |> signupButtonStyle
   }
 
-  internal override func bindViewModel() {
-    self.emailTextField.rac.becomeFirstResponder = self.viewModel.outputs.emailTextFieldBecomeFirstResponder
-    self.newsletterSwitch.rac.on = self.viewModel.outputs.setWeeklyNewsletterState
-    self.passwordTextField.rac.becomeFirstResponder =
-      self.viewModel.outputs.passwordTextFieldBecomeFirstResponder
-    self.signupButton.rac.enabled = self.viewModel.outputs.isSignupButtonEnabled
+  fileprivate let environmentLoggedInProperty = MutableProperty(())
+  fileprivate let viewDidLoadProperty = MutableProperty(())
 
-    self.viewModel.outputs.logIntoEnvironment
+  internal override func bindViewModel() {
+    let (
+      emailTextFieldBecomeFirstResponder,
+      isSignupButtonEnabled,
+      logIntoEnvironment,
+      passwordTextFieldBecomeFirstResponder,
+      postNotification,
+      nameTextFieldBecomeFirstResponder,
```

Comment:
Turns out this output isn't hooked up! If you build my branch you'll see an unused ⚠️ here. A good opportunity to hook it up or delete it if it's no longer needed 😄